site.yml

```diff
@@ -70,6 +70,7 @@
   - { role: contiv_storage, run_as: worker }
 
 # netplugin-node hosts set up netmast/netplugin in a cluster
+# Set aci_mode to true to run node in ACI mode
```

Comment:
related to previous comment, this documentation shall be part of the networking mode variable in `group-vats/all` file